### DISCUSSION TITLE
Whether PIXI supports quick preview window (partial window)

### DISCUSSION THREAD

**Discussion by pcbren1987** [pixijs/pixijs#8356] [Q&A]
Drawing PCB with PIXI, there are many graphics, need to add preview window, you can enlarge the partial graphics display, if using two PIXI instances will be slow, is there a faster way?
Whether PIXI has the function of local graph projection?
As shown in the figure below

![image](https://user-images.githubusercontent.com/89827437/169631084-873bdc8e-7629-45d4-9b20-649f367f55da.png)

**Comment by domis86** [comment]
( linking similar discussion: https://github.com/pixijs/pixijs/discussions/8567 )
